### DISCUSSION TITLE
begginer questions...

### DISCUSSION THREAD

**Discussion by evpUK** [open-mmlab/mmpose#2927] [General]
Hi there, I am newbie and I have a few questions maybe someone can help me with :

I am trying to create a simple app that extracts a 2d skeleton from a still image and then exports the found keypoints to a JSON file, I am going to create a custom skeleton and data set in v7 labs Darwin and train a model that pinpoints where I want the joints centers to arrive based on a specific motion capture actor calibration workflow we are using. I found MMpose and it seems to be a sensible platform for doing this. So after looking at the image demo its already doing more or less what I need just with a different skeleton and no JSON output, so I have some questions as I am finding the docs fairly overwhelming for beginners.

1. Regarding the image demo..  how could I change the model being used for the image demo to a different model such as the JHMBD model? I would later intend to replace this with my own and so I want to know how to use a different one.
(I have already downloaded it and placed the dataset and annotations as advised)

3. If I create my own dataset and annotations using v7 labs Darwin platform should I export it to COCO format for ease of use? 

4. Could I train it on that platform and then add the files to MMpose or do I need to train it in MMpose (to create the train.JSON)? 

5. Does MMpose allow for writing of the keypoint co-ordinates generated from the model to a JSON file? if so how do I do this..

Big thanks to anyone who can help me with this!

cheers :)

Alex

**Comment by Ben-Louis** [comment]
To change the model used in the demo command, you need to modify the config and checkpoint files. When you use the `--save-predictions` argument, the program will save the inference results in a JSON file.
